src/types/modifications/ReserveRequirement.jl

```diff
@@ -47,7 +47,18 @@ Representation of reserve requirement, such that the sum of eligible power injec
   * `cons_pres_<name>` - (nsubarea x nyr x nhr) Constrain that `pres_total_subarea_<name> ≥ pres_req_subarea_<name>`.
 
 ### Results Formulas:
+* `(:<name>_requirements, :pres_req)` - the hourly required reserve power for the region.
+* `(:<name>_requirements, :pres_supply)` - the hourly supplied reserve power for the region. 
+* `(:<name>_requirements, :eres_supply)` - the supply for reserve energy. May be misleading but useful for calculations. 
+* `(:<name>_requirements, :cost_total)` - cost paid for the supplied reserve power. 
+* `(:<name>_requirements, :price_avg)` - the total cost divided by the supply of reserve power. This represents the shadow price on the capacity constraint, in dollars per MW reserve capacity supplied per hour. 
+* `(:<name>_requirements, :pres_flow)` - the hourly power flowing out of the region. May be misleading when aggregating above the subarea level. 
+* `(:bus, :<name>_cost_result)` - the total rebate paid by users to EGUs (and storage if present) from the $(mod.name) reserve requirement, not including merchandising surplus.
```

Comment:
The dollar sign inside of a string in julia represents string interpolation.  For example:
```julia
julia> a = 5
5
julia> s = "this is a string and here is a: $(a)"
"this is a string and here is a: 5"
```

You'll need to remove all the references to `$(mod.name)` from the docstring.  You can probably just remove them or replace with "the reserve requirement" or similar.